core/templates/domain/exploration/InteractionObjectFactory.ts

```diff
@@ -31,11 +31,12 @@ import { ISolutionBackendDict, Solution, SolutionObjectFactory } from
   'domain/exploration/SolutionObjectFactory';
 import { IInteractionCustomizationArgs } from
   'interactions/customization-args-defs';
+import { IInteractionAnswer } from 'interactions/answer-defs';
 
 export interface IInteractionBackendDict {
   'default_outcome': IOutcomeBackendDict;
   'answer_groups': IAnswerGroupBackendDict[];
-  'confirmed_unclassified_answers': any;
+  'confirmed_unclassified_answers': IInteractionAnswer;
```

Comment:
Name suggests that this is an array? or is it not?

Reply:
yeah, right.